tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -288,8 +289,7 @@ tuple<
     optional<std::unordered_map<std::string, shared_ptr<ArraySchema>>>>
 StorageManagerCanonical::array_open_for_reads_without_fragments(Array* array) {
   auto timer_se =
-      stats_->start_timer("sm_array_open_for_reads_without_fragments");
-
+      stats_->start_timer("array_open_read_without_fragments_load_schemas");
```

Comment:
If I'm not mistaken, this changes an existing stat name which I've been generally treating as "breaking public API changes". I've got no idea if we have a policy like that for these JSON blobs or not. Granted, I do prefer the new name here as it seems more descriptive.

Reply:
@ihnorton @Shelnutt2 Do we consider stats name changes as public API changes?

Reply:
We do not currently have any policy/guarantees about the stats output format, and I think the only policy that is feasible right now is name stability within a minor release version. Please make sure to include it as a `TYPE: BREAKING` in the PR description so that we can highlight the change in the release notes.

Reply:
Thanks @ihnorton , done.